src/components/expanded-state/AvailableNetworksv2.tsx

```diff
@@ -0,0 +1,230 @@
+        <Box
+          as={ButtonPressAnimation}
+          // @ts-expect-error overloaded props ButtonPressAnimation
+          scaleTo={0.96}
+          onPress={availableNetworks.length === 1 ? handlePressButton : NOOP}
+          marginHorizontal={{ custom: marginHorizontal }}
+        >
+          <Box
+            borderRadius={99}
+            padding="8px"
+            paddingHorizontal={{ custom: 12 }}
+            justifyContent="center"
+          >
+            <RadialGradient
+              {...radialGradientProps}
+              // @ts-expect-error overloaded props RadialGradient
+              borderRadius={99}
+              radius={600}
+            />
+            <Inline alignVertical="center" alignHorizontal="justify">
+              <Inline alignVertical="center">
+                <Box style={{ flexDirection: 'row' }}>
+                  {availableNetworks?.map((network, index) => {
+                    return (
+                      <Box
+                        background="body"
+                        key={`availableNetwork-${network}`}
+                        marginLeft={{ custom: -4 }}
+                        style={{
+                          backgroundColor: colors.transparent,
+                          zIndex: availableNetworks?.length - index,
+                          borderRadius: 30,
+                        }}
+                      >
+                        {network !== 'mainnet' ? (
+                          <ChainBadge
+                            assetType={network}
+                            position="relative"
+                            size="small"
+                          />
+                        ) : (
+                          <CoinIcon
+                            address={ETH_ADDRESS}
+                            size={20}
+                            symbol={ETH_SYMBOL}
+                            forcedShadowColor={undefined}
+                            type={AssetType.token}
+                          />
+                        )}
+                      </Box>
+                    );
+                  })}
+                </Box>
+
+                <Box paddingLeft="6px">
+                  <Text
+                    color="secondary60"
+                    size="14px"
+                    weight="semibold"
+                    numberOfLines={2}
+                  >
+                    {availableNetworks?.length > 1
+                      ? lang.t('expanded_state.asset.available_networks', {
+                          availableNetworks: availableNetworks?.length,
+                        })
+                      : lang.t('expanded_state.asset.available_networkv2', {
+                          availableNetwork:
+                            networkInfo[availableNetworks?.[0]]?.name,
+                        })}
+                  </Text>
+                </Box>
+              </Inline>
+              <Text
+                align="center"
+                color="secondary60"
```

Comment:
https://www.figma.com/file/SFfYzRoHlapEecS0LTLwhQ/Swap-Aggregator-V2?node-id=898%3A56903 
i believe is `secondary40`?

Reply:
figured they'd be the same 👌